spec/system/custom_sidebar_sections_spec.rb

```diff
@@ -190,7 +196,21 @@
     expect(page).not_to have_button("Edited public section")
   end
 
+  it "shows anonymous public sections" do
+    sidebar_section = Fabricate(:sidebar_section, title: "Public section", user: user, public: true)
+    sidebar_url_1 = Fabricate(:sidebar_url, name: "Sidebar Tags", value: "/tags")
+    Fabricate(:sidebar_section_link, sidebar_section: sidebar_section, linkable: sidebar_url_1)
+    sidebar_url_2 = Fabricate(:sidebar_url, name: "Sidebar Categories", value: "/categories")
+    Fabricate(:sidebar_section_link, sidebar_section: sidebar_section, linkable: sidebar_url_2)
+
+    visit("/latest")
+    expect(page).to have_button("Public section")
```

Comment:
```suggestion
    expect(sidebar).to have_section("Public section")
```

What do you think about having something like this instead? Technically a button that is outside the sidebar with the text "Public section" will still cause this assertion to pass.